htp/htp_response.c

```diff
@@ -550,21 +550,29 @@ htp_status_t htp_connp_RES_BODY_DETERMINE(htp_connp_t *connp) {
         }
     }
 
+    htp_header_t *ct = htp_table_get_c(connp->out_tx->response_headers, "content-type");
```

Comment:
ct doesn't look like it's used before the old location of invocation, so it's probably best to move it back? These table lookups are not free, so best to only do them as needed.

Reply:
Ok, well spotted